### PR TITLE
SAMOA-17: Update info in README.md in master branch

### PR DESCRIPTION
**Pull request by abifet**

### CODE REVIEW

**Comment by gdfm** [comment]
I think we should keep the license header.
I would also keep the build badge on Travis CI, just update it to the new build URL:
https://travis-ci.org/apache/incubator-samoa.svg?branch=master

File: README.md [hunk 1]
```diff
@@ -60,35 +43,35 @@ mvn -Ps4 package
...
+[Javadoc](http://samoa.incubator.apache.org/docs/api/)
 
 ## Slides
 
-[![SAMOA Slides](http://yahoo.github.io/samoa/samoa-slides.jpg)](https://speakerdeck.com/gdfm/samoa-a-platform-for-mining-big-data-streams)
+[![SAMOA Slides](http://samoa.incubator.apache.org/samoa-slides.jpg)](https://speakerdeck.com/gdfm/samoa-a-platform-for-mining-big-data-streams)
 
-G. De Francisci Morales [SAMOA: A Platform for Mining Big Data Streams](http://melmeric.files.wordpress.com/2013/04/samoa-a-platform-for-mining-big-data-streams.pdf)
-Keynote Talk at [RAMSS ’13](http://www.ramss.ws/2013/program/): 2nd International Workshop on Real-Time Analysis and Mining of Social Streams WWW, Rio De Janeiro, 2013.
+G. De Francisci Morales, A. Bifet [SAMOA: A Platform for Mining Big Data Streams](http://jmlr.csail.mit.edu/papers/volume16/morales15a/morales15a.pdf)
```
Comment (gdfm) [inline README.md]: The title should be "SAMOA: Scalable Advanced Massive Online Analysis"

**Comment by abifet** [comment]
I did the changes: update license, travis info and journal title.

**Comment by gdfm** [comment]
+1
Please squash the commits into a single one for the issue when you merge it.

**Pull request closed without merging**